node/src/components/storage.rs

```diff
@@ -1269,8 +1269,6 @@ impl Storage {
                 {
                     continue;
                 }
-                // todo!() - right deploy with incorrect approvals found in DB, download and
```

Comment:
Wouldn't we still want to download the correct approvals in this case?

Reply:
I think that's what would happen when the node falls back to leaping and gets this block via historical sync.